api/v1/models/UniversalTrackingItem.js

```diff
@@ -0,0 +1,17 @@
+var Model = require('./Model');
+
+var UniversalTrackingItem = Model.extend({
+    tableName: 'event_tracking',
+    idAttribute: 'id',
+    hasTimestamps: ['created'],
+    validations: {
```

Comment:
Just at a quick glance, I think start time would make things a bit tricker. We could probably do end time, but without start time that kinda means the same thing as duration. Maybe @redsn0w422 you can add start/end time as an issue (with tag `enhancement`) and we'll do it if there's time.